spec/parse_date_spec.rb

```diff
@@ -10,7 +10,7 @@
       # single value
       { # string to parse as key, expected result as value
         '12/25/00' => [2000],
-        '5-1-25' => [1925],
+        '5-1-25' => [2025],
```

Comment:
This is causing a test failure that may or may not be a regression. If we're concerned, I'm happy to ticket/investigate separately.